pkg/terraform/exec/plugins/Gopkg.lock

```diff
@@ -798,6 +798,22 @@
   revision = "c68a8212f13053ab7b45f66f7c6deded348bcd23"
   version = "v0.1.7"
 
+[[projects]]
```

Comment:
this probably needs to be split into separate commit from `ovirt: Implement gather`